src/ert/_c_wrappers/enkf/enkf_fs_manager.py

```diff
@@ -102,10 +102,16 @@ def __getitem__(self, case_name: str) -> EnkfFs:
         if case_name in self.open_storages:
             return self.open_storages[case_name]
         elif case_name in self.cases:
-            return self.add_case(case_name)
+            file_system = EnkfFs.createFileSystem(
+                self.storage_path / case_name,
+                self._ensemble_config,
+                self._ensemble_size,
+                self.read_only,
+            )
+            self._add_to_open(file_system)
+            return file_system
         else:
             raise KeyError(f"No such case name: {case_name} in {self.cases}")
 
-    def umount(self) -> None:
```

Comment:
Was this ever used?

Reply:
Yes, I wrote it in the commit body, this was used when some storages were owned by python and some by C, but now all are owned by python and are QC'ed automatically